app/models/declaration/submissions/Submission.scala

```diff
@@ -21,17 +21,19 @@ import models.declaration.submissions.RequestType.{CancellationRequest, Submissi
 import play.api.libs.json.Json
 
 import java.time.ZonedDateTime
-import java.util.UUID
 
 case class Submission(
-  uuid: String = UUID.randomUUID.toString,
+  uuid: String,
   eori: String,
   lrn: String,
   mrn: Option[String] = None,
   ducr: Option[String] = None,
   latestEnhancedStatus: Option[EnhancedStatus] = None,
   enhancedStatusLastUpdated: Option[ZonedDateTime] = None,
-  actions: Seq[Action]
+  actions: Seq[Action],
+  latestDecId: String,
+  latestVersionNo: Int = 1,
```

Comment:
Nit: is this default needed? All new submission entities are created on the BE no?

Reply:
It facilitates creating instances of `Submission` in our unit tests.